common/thorhelper/roxiehelper.cpp

```diff
@@ -1555,7 +1555,21 @@ void IRoxieContextLogger::CTXLOGae(IException *E, const char *file, unsigned lin
 
 void HttpHelper::gatherUrlParameters()
 {
-    const char *finger = strchr(urlPath, '?');
+    const char *start = urlPath.str();
+    while (isspace(*start))
```

Comment:
I'll reorganize.  Btw, my goal is to slowly evolve this into roxie supporting calling queries with HTTP-GET (i.e. URL with parameters).